src/app/components/ErrorDisplay/index.tsx

```diff
@@ -65,7 +78,22 @@ const errorMap: Record<AppErrors, (t: TFunction, error: ErrorPayload) => Formatt
     title: t('errors.notFoundProposal'),
     message: t('errors.validateURL'),
   }),
-  [AppErrors.InvalidUrl]: t => ({ title: t('errors.invalidUrl'), message: t('errors.validateURL') }),
+  [AppErrors.InvalidUrl]: (t, _, scope, theme) => ({
+    title: t('errors.invalidUrl'),
+    message: (
+      <Box
+        sx={{
+          a: { color: theme.palette.layout.contrastMain, textDecoration: 'underline' },
+        }}
+      >
+        <p>
+          <SearchSuggestionsLinksForNoResults scope={scope} suggestSearch />
+        </p>
+        <p>{t('errors.alternativelyGoExplore')}</p>
+        <ExploreOasisButton />
+      </Box>
```

Comment:
How come this lives here? (other errors seem to have custom styling outside this general display)